src/components/ContactUsForm/index.jsx

```diff
@@ -35,8 +38,12 @@ const ContactUsForm = () => {
       headers: { "Content-Type": "application/x-www-form-urlencoded" },
       body: encodeFormData({ "form-name": "contact", ...formValue }),
     }).then(() => {
-      setTimeout(() => setIsSubmitted(false), 2000);
+      setTimeout(() => {
```

Comment:
as mentioned you should never use settimeout, i shouldn't have let this past code review so that was my mistake

you are assuming the contact form will take < 2 seconds (you sohuld never make any assumptions about time it takes for something to happen like this)

you want it to do "setisLoading" before you call the fetch, under line 34.

then you want it to setIsLoading to false in the ".then"

you probably want to delete the line about setIsSubmitted(false). because you don't really want the submission msg to go away after the user does it. or use antd.message

Reply:
Thanks for reviewing, I tried following the steps as you have mentioned, PTAL.